test/cli.ts

```diff
@@ -31,16 +46,12 @@ describe('main', () => {
       './handle-git-dir-change': stubHelperHandlers,
     });
     stubWorkFlow.main();
-    assert.isOk(true);
   });
   it('fails when there is no env variable', async () => {
-    try {
-      sandbox.stub(process.env, 'ACCESS_TOKEN').value(undefined);
```

Comment:
Another fun fact!  This code was throwing when it tried to create the stub, because `ACCESS_TOKEN` didn't exist as a property.  This test was bunk.